### PR TITLE
Better formatting?

### PR DESCRIPTION
**Pull request by dvdplm**
Add a rustfmt.toml and set max_width = 120 (default: 100) and use_small_heuristics = "Max"

I made a few variants for comparison:

- [#31c9c8d](https://github.com/entropyxyz/crypto-primes/pull/51/commits/31c9c8d6c175b0e954db2ae8c93365d13e318c91): `max_width = 120` + `use_small_heuristics = "Max"`
- https://github.com/entropyxyz/crypto-primes/pull/51/commits/3beec30351786fe32533e7af559ebd703010a77f: `max_width = 120` + `use_small_heuristics = "Default"`
- https://github.com/entropyxyz/crypto-primes/pull/51/commits/afcf61d29ceb13d7407f9007ab3bbe8eece7b35a: `max_width = 100` + `use_small_heuristics = "Max`

I committed my personal preference as the last one: 120 width and default heuristics.

### CODE REVIEW

File: rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+max_width = 120
+use_small_heuristics = "Default"
```
Comment (fjarri) [inline rustfmt.toml:2]: Probably no need to specify it if it's the default.

**Comment by fjarri** [comment]
Yes, of these three I think I prefer the 120 width and default heuristics variant too.

I have some reluctance towards going against the all default settings, but line length is really the least obtrusive one.

**Comment by dvdplm** [comment]
We should probably merge this when the other PRs are all merged.

**Comment by fjarri** [comment]
Ok, let's do that.

**Pull request merged**